payjoin/src/uri.rs

```diff
@@ -125,8 +210,43 @@ impl From<InternalPjParseError> for PjParseError {
     fn from(value: InternalPjParseError) -> Self { PjParseError(value) }
 }
 
+impl<'a> bip21::SerializeParams for &'a MaybePayjoinExtras {
+    type Key = &'static str;
```

Comment:
why this one is not using the 'a lifetime from the signature?

Reply:
the keys ("pj", "pjos", "ohttp) are known at compile time. I saw no reason to restrict them to 'a.